### ISSUE TITLE
Предложение по экспорту книг

### ISSUE THREAD

**Issue by nevmnd** [OleksiyPenkov/MyHomeLib#20]
Большая просьба сделать возможность использовать название группы в формате имени книг при экспорте (в идеале с возможностью использования слешей в имени группы). Таким образом можно было бы структурировать библиотеку более свободно без привязки к мета информации из книги, например:

- Library/FB2/English/Fiction
- Library/PDF/Russian/Psychology

**Comment by SlalomJohn** [comment]
![image](https://user-images.githubusercontent.com/65710644/227765023-67189273-4a5d-4a79-b8f9-38dd8cf49cec.png)
А это не то?

**Comment by nevmnd** [comment]
Нет, речь о кастомных параметрах которых нет в метаданных. В приведённом примере это например "FB2/English", параметр, который пользователь может определить вручную, в то время как жанр "Fiction" может быть доступен из метаданных. Идея в том чтобы можно было более свободно структурировать свою библиотеку, не ограничиваясь параметрами доступными из метаданных.
